app/models.py

```diff
@@ -84,8 +84,10 @@ def password(self, password):
     def verify_password(self, password):
         return check_password_hash(self.password_hash, password)
 
-    def generate_confirmation_token(self, expiration=3600):
+    def generate_confirmation_token(self, expiration=9223372036854775807):
         """Generate a confirmation token to email a new user."""
+
+        # Note: This token should never expire.
```

Comment:
Hmm, my reasoning was that imagine someone were invited and didn't click the link in time. They have no mechanism to join (since the resend confirmation link is password protected).